docs/InstallGuide.md

```diff
@@ -1,424 +1,358 @@
+  * openvnet-common
+  * openvnet-vna
+  * openvnet-vnmgr
+  * openvnet-webapi
 
+* openvnet-third-party.repo
+  * openvnet-ruby
+  * openvswitch
 
-OpenVNet installation
-------------------------
+Install epel-release.
+
+    # yum install -y epel-release
 
-Install OpenVNet.
+
+Install OpenVNet packages.
 
     # yum install -y openvnet
 
-Initial Configuration
----------------------
+`openvnet` is a metapackage. It is equivalent to installing `openvnet-common`,
+`openvnet-vna`, `openvnet-vnmgr` and `openvnet-webapi` at once.
 
-### Setup vnmgr
+### Edit Configuration Files
 
-Modify the following section in /etc/openvnet/vnmgr.conf on the vnmgr node.
+Edit the file `/etc/openvnet/vnmgr.conf`
 
     node {
       id "vnmgr"
       addr {
         protocol "tcp"
-        host "192.168.100.2"
+        host "127.0.0.1"
+        public ""
         port 9102
       }
     }
 
-- **host** : The IP address or host name of the ZeroMQ node.
-- **port** : The port number for connecting with ZeroMQ. If the vnmgr process is run on the same node as vna or webapi, specify a different value from vna and webapi.
+Modify the parameters `host` and `public` according to your environment. In order for
+the sample environment in the overview section we leave those parameters as is. The detail of each parameter is following.
 
+- **id** : OpenVNet uses the [0mq](http://zeromq.org) protocol for communication between its processes. This id is used by 0mq to identify each process. Any string here is fine as long as there's no two processes using the same one. It's recommended to just use the default values.
 
-### Setup vna
+- **protocol** : The network protocol that the OpenVNet's processes will use for communication. The default `tcp`.
 
-Modify the following section in /etc/openvnet/vna.conf on the node.
+- **host** : The IP address of the vnmgr node. Since all the processes reside on the same node we use loopback address in this guide.
 
-    node {
-      id "vna"
-      addr {
-        protocol "tcp"
-        host "192.168.100.2"
-        port 9103
-      }
-    }
+- **public** : In case the process running in a NAT environment, specify the NAT address as the process can advertise its location.
 
-- **host** : The IP address or host name of the ZeroMQ node.
-- **port** : The port number for connecting with ZeroMQ. If the vna process is run on the same node as vnmgr or webapi, specify a different value from vnmgr and webapi.
+- **port** : The port number that the process will listen to. Specify a unique port number and make sure the port number is different for each of the OpenVNet's processes and also not taken by any other process.
 
+`/etc/openvnet/vna.conf` and `/etc/openvnet/webapi.conf` have the same structure as `vnmgr.conf`. Edit them if necessary otherwise leave them as is for the sample environment we are just creating. However the `id` parameter in `vna.conf`, we need it for the database configuration so please remember what you specified.
```

Comment:
I think the last sentence is a little hard to read. This would be better.

"We will need the `id` parameter in `vna.conf` later when we configure the database. Make sure you remember what you specified."